spec/values/calculation/round/one_argument.hrx

```diff
@@ -52,6 +63,27 @@ a {
   b: 2px;
 }
 
+<===> preserves_single_unit/warning
+DEPRECATION WARNING: In future versions of Sass, round() will be interpreted as the CSS round() calculation. This doesn't allow unitless numbers to be mixed with numbers with units. If you want to use the Sass function, call math.round() instead.
+
+See https://sass-lang.com/d/import
+
+  ,
+1 | a {b: round(1 + 1px)}
+  |             ^^^^^^^
+  '
+    input.scss 1:13  root stylesheet
+
+DEPRECATION WARNING: In future versions of Sass, round() will be interpreted as a CSS round() calculation. This requires an explicit modulus when rounding numbers with units. If you want to use the Sass function, call math.round() instead.
```

Comment:
Is a double warning expected here?

Reply:
More or less. It's both the case that `round(1 + 1px)` will throw an error *and* that `round(2px)` (which it resolves to) will throw an error.